app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render partial: "stacks/header", locals: { stack: @stack } %>
+
+<div class="wrapper"
+  <section>
+    <ul class="commit-list">
+      <%= render @tasks, read_only: true %>
+    </ul>
+  </section>
+</div>
+
+<div class="wrapper"
+  <section>
+    <% if params[:since].present? %>
+      <%= link_to 'Start', @links[:first], class: 'header__btn btn' %>
+    <% else %>
+      <%= link_to 'Start', '', class: 'header__btn btn btn--disabled disabled' %>
+    <% end %>
+
+    <% if url = @links[:next] %>
```

Comment:
This line give a ruby warning. I hate this warning too but it is a good practice to write warning free code. To do so put the `url = @links[:next]` inside parentheses